docs/node-operators/seed-peers/index.mdx

```diff
@@ -7,6 +7,15 @@ keywords:
 ---
 
 
-# Seed Peers
+# Introduction
 
-Seed peers are our helpful allies that make it easy for the rest of us to connect to the network. Most nodes connect to the Mina network using seed peers that are run explicitly as seed nodes.
\ No newline at end of file
+Seed Peer Providers are an independent group of nodes that assist new nodes to find peers and connect to the Mina Protocol. This improves the onboarding experience for other nodes, and makes sure they start bootstrapping as soon as possible.
+
+Mina’s Seed Peers are considered to be leaders amongst the community. They are not compensated in any way – they perform their role as a service to the Mina community. The full list of seed peers can be found [here](https://minaprotocol.com/ja/blog/seed-peer-providers-program)
```

Comment:
```suggestion
Mina’s seed peers are considered to be leaders among the community. They are not compensated in any way – they perform their role as a service to the Mina community. For the full list of seed peers, see [Seed Peer Providers Program](https://minaprotocol.com/blog/seed-peer-providers-program).
```

Reply:
oh interesting, the Japanese language was embedded in the URL, so I saw the page in Japanese! I removed the `/ja` from the URL